1_4sections.tex

```diff
@@ -30,52 +30,52 @@
+	\item \verb|CompReport.sso|: Observed and expected composition data in a list-based format.
+	\item \verb|Forecast-report.sso|: Output of management quantities and for forecasts.
+	\item \verb|CumReport.sso|: This file contains a brief version of the run output, output is appended to current content of file so that the results of several runs can be collected together. This is useful when a batch of runs is being processed.
+	\item \verb|Covar.sso|: This file replaces the standard \gls{admb} \verb|ss.cor| with an output of the parameter and derived quantity correlations in database format.
+	\item \verb|ss3.par| (previously \verb|ss.par|): This file contains all estimated and fixed parameters from the model run. 
+	\item \verb|ss.std|, \verb|ss.rep|, \verb|ss.cor|, etc.: Standard \gls{admb} output files.
+	\item \verb|checkup.sso|: Contains details of selectivity parameters and resulting vectors. This is written during the first call of the objective function.
+	\item \verb|Gradient.dat|: New for v.3.30, this file shows parameter gradients at the end of the run.
+	\item \verb|rebuild.dat|: Output formatted for direct input to Andre Punt's rebuilding analysis package. Cumulative output is output to REBUILD.SS (useful when doing \gls{mcmc} or profiles).
+	\item \verb|SIS\_table.sso|: Output formatted for reading into the \gls{nmfs} \href{https://www.st.nmfs.noaa.gov/sis/}{\gls{sis}}.
+	\item \verb|Parmtrace.sso|: Parameter values at each iteration.
+	\item \verb|posteriors.sso|, \verb|derived\_posteriors.sso|, \verb|posterior\_vectors.sso|: Files associated with \gls{mcmc}.
+\end{enumerate}
 
 \pagebreak
 
 \hypertarget{StartingSS3}{}
 \section[Starting Stock Synthesis]{\protect\hyperlink{StartingSS3}{Starting Stock Synthesis}}
-SS3 is typically run through the command line interface, although it can also be called from another program, R, the Stock Synthesis Interface, or a script file (such as a DOS batch file). SS3 is compiled for Windows, Mac, and Linux operating systems. The memory requirements depend on the complexity of the model you run, but in general, SS3 will run much slower on computers with inadequate memory. See \hyperref[sec:RunningSS3]{Running Stock Synthesis} for additional notes on methods of running SS3.
+SS3 is typically run through the command line interface, although it can also be called from another program, R, the \gls{sis}, or a script file (such as a DOS batch file). SS3 is compiled for Windows, Mac, and Linux operating systems. The memory requirements depend on the complexity of the model you run, but in general, SS3 will run much slower on computers with inadequate memory. See \hyperref[sec:RunningSS3]{Running Stock Synthesis} for additional notes on methods of running SS3.
```

Comment:
If "SIS" is pronounced es i es then keep "the SIS" but if it is pronounced "sis" like sister then remove "the".

Reply:
Should be SSI anyways. Fixed.